mantidimaging/gui/test/gui_system_test.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (C) 2021 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+
+import os
+from pathlib import Path
+import sys
+import unittest
+from unittest import mock
+
+from PyQt5.QtCore import Qt, QTimer
+from PyQt5.QtTest import QTest
+from PyQt5.QtWidgets import QApplication, QMessageBox
+import pytest
+
+from mantidimaging.core.utility.version_check import versions
+from mantidimaging.gui.windows.main import MainWindowView
+from mantidimaging.gui.windows.load_dialog.presenter import Notification
+
+versions._use_test_values()
+
+LOAD_SAMPLE = str(Path.home()) + "/mantidimaging-data-master/ISIS/IMAT/IMAT00010675/Tomo/IMAT_Flower_Tomo_000000.tif"
```

Comment:
I recommend placing the test image in a **test_data** folder which should be in the repo

Reply:
We currently keep it outside the main repo because it is about 600MB and has different licensing. But I agree that requiring it to live in the users home directory is not ideal. I'll have a think.